backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/CassandraCluster.java

```diff
@@ -92,7 +92,15 @@ public void clearAllTables() {
     }
 
     public Cluster getCluster() {
-        return ClusterFactory.createTestingCluster(CLUSTER_IP, CLUSTER_PORT_TEST);
+        return ClusterFactory.builder()
+                .host(CLUSTER_IP)
+                .port(CLUSTER_PORT_TEST)
+                .queryLogger(
+                    ClusterFactory.QueryLoggerConf.builder()
+                    .withConstantThreshold(500)
```

Comment:
> Should I introduce a constant for 500 or should I set it somewhere in a property file.

Yes please. Constant is the minimum. Conf file with default value to constant is better

> By the way which value should I use or should I use [dynamic thresholds](Dynamic thresholds). IMO I think dynamic thresholds should be used in prod and constant for test.

What's the rationals behind it ?